staff/gerard-celma/cv/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>CV</title>
+    </head>
+    <body>
+        <header>
+            <h1>Waldo Geraldo Faldo</h1>
+            <img src="">
+            <address>street, postcode barcelona, spain</address>
+            <p>Telephone: 666 66 66 66</p>
+            <p>Email: waldo@geraldofaldo.com</p>
+        </header>
+        <main>
+            <section id="about me">
```

Comment:
You can set space between the words, you should write `about_me` or camelcase `aboutMe`

Reply:
Sorry I mean can't

Reply:
fixed, thanks!